packages/lodestar/src/sync/reqResp/reqResp.ts

```diff
@@ -64,12 +65,18 @@ export class SyncReqResp implements ISyncReqResp {
   }
 
   public async start(): Promise<void> {
+    // TODO: enable after we add peerbook persisting
```

Comment:
what does this mean?

Reply:
Well curently, when this code is run we don't have any peers since peerbook is wiped clean onstartup. I guess in future we can store peers in database or on disk and when starting we can try to reconnect them and fetch their latestStatus.

I should probably uncomment this as it's gonna work as soon as we have that! :+1: